v3/src/components/data-display/models/data-display-content-model.ts

```diff
@@ -43,6 +43,14 @@ export const DataDisplayContentModel = TileContentModel
     hasDraggableNumericAxis(axisModel: IAxisModel): boolean {
       return isNumericAxisModel(axisModel) && self.pointDisplayType !== "bins"
     },
+    hasHistogramAxis() {
+      // derived models should override
+      return false
+    },
+    histogramAxisTicks(): { tickValues: number[], tickLabels: string[] } {
+      // derived models should override
+      return {tickValues: [], tickLabels: []}
+    },
```

Comment:
Another thing I'm especially wondering about is the names for these methods. It seems to me that the histogram does require a special numeric axis where the ticks are equivalent to the bin boundaries in the binned plot. They are different from the binned plot's axis (which uses `nonDraggableAxisTicks`) in that the axis should be draggable and have different tick values. These methods get used in the Axis world, though, so should probably have less graph-centric names? I haven't been able to think of anything better, though.

Reply:
As we discussed on Slack, the histogram's numeric axis is perfectly ordinary, just providing a mapping from "world" values to "pixel" values.